rules/container/metadata_merge.bzl

```diff
@@ -0,0 +1,85 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+metadata_merge accepts a list of YAML metadata generated by the download_pkgs
+rule & packages_metadata merges their contents into a single YAML file.
```

Comment:
& uses packages_metadata to merge ...

Reply:
Broke down this statement into two to clarify it's metadata_merge that does the merging, not packages_metadata.